### ISSUE TITLE
`axis.line.x` and `axis.line.y` work, but not `axis.line`

### ISSUE THREAD

**Issue by johnbaums** [hadley/ggplot2#1567]
In 2.1.0, `axis.line.x` and `axis.line.y` work, but `axis.line` does not.

For example, 

This works:
```r
ggplot(mtcars, aes(wt, mpg)) +
  geom_point() +
  theme(axis.line.x=element_line(),
        axis.line.y=element_line())
```

![image](https://cloud.githubusercontent.com/assets/818438/13516186/f039a844-e20b-11e5-917c-ea2441672b42.png)

But this doesn't:
```r
ggplot(mtcars, aes(wt, mpg)) +
  geom_point() +
  theme(axis.line=element_line())
```

![image](https://cloud.githubusercontent.com/assets/818438/13516191/fcace80c-e20b-11e5-8764-b85c23c102db.png)

(The example at http://docs.ggplot2.org/current/theme.html - i.e., `m + theme(axis.line = element_line(size = 3, colour = "red", linetype = "dotted"))` - also fails)

**Comment by amoeba** [comment]
If it helps, I can confirm that I am seeing this behavior as well.

Using `ggplot2    * 2.1.0    2016-03-01 CRAN (R 3.2.3) ` Mac OS 10.11.3. Seems like this could be related to #778.

**Comment by NattyE** [comment]
I believe this is also an issue with `panel.grid`, as this will work for `panel.grid.major` and `panel.grid.minor` but not with `panel.grid`.

**Comment by GrosPhoton** [comment]
The same problem seems to happen with `line`

**Comment by Katiedaisey** [comment]
@johnbaums @NattyE @GrosPhoton This has been solved in pull request #1581 that has unfortunately not been integrated into the recent ggplot2 update. You can access it there or directly from my branch of ggplot2.

**Comment by thomasp85** [comment]
Fixed in #1754
